support-frontend/build-tc

```diff
@@ -3,6 +3,12 @@
 # Exit if any of these commands fail
 set -e
 
+export NVM_DIR="$HOME/.nvm"
```

Comment:
This is unrelated to the change. Teamcity's nvm step has stopped working, so we're doing it in the build script instead